Mondee Watch App/View/BeforeGameView.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  BeforeGameView.swift
+//  Mondee Watch App
+//
+//  Created by 하명관 on 2023/07/14.
+//
+
+import SwiftUI
+
+struct BeforeGameView: View {
+    
+    // MARK: Properties
+    
+    @Binding var mondeeScroll: CGFloat
+    @Binding var isGuideActive: Bool
+    
+    private let topScrollLimit = 250
+    private let mondeePadding = 100
+    private let mondeeFinalLocationRatio = 2.5
+    
+    let bottomScrollLimit : CGFloat
+    
+    var body: some View {
+        GeometryReader { geo in
+            
+            let deviceHeight = geo.size.height
+            let deviceWidth = geo.size.width
+            let mondeeWidth = geo.size.width - CGFloat(mondeePadding)
+            
+            NavigationView {
+                ZStack(alignment: .top) {
+                    
+                    // MARK: MONDEE
+                    
+                    VStack() {
+                        Image("ImageWatchMondee")
+                            .resizable()
+                            .scaledToFit()
+                            .frame(maxWidth: mondeeWidth)
+                            .offset(y: startGame ? deviceHeight/mondeeFinalLocationRatio : -((self.mondeeScroll)/6))
+                            .focusable(isFocusable)
+                            .digitalCrownRotation($mondeeScroll, from: CGFloat(bottomScrollLimit), through:CGFloat(topScrollLimit))
+                    }
```

Comment:
```suggestion
                    Image("ImageWatchMondee")
                        .resizable()
                        .scaledToFit()
                        .frame(maxWidth: mondeeWidth)
                        .offset(y: startGame ? deviceHeight/mondeeFinalLocationRatio : -((self.mondeeScroll)/6))
                        .focusable(isFocusable)
                        .digitalCrownRotation($mondeeScroll, from: CGFloat(bottomScrollLimit), through:CGFloat(topScrollLimit))
```

일단 VStack은 빼야할 것 같고,
먼디 캐릭터를 구조체로 만들면 먼디의 모습, 상태 등을 용이하게 관리할 수 있을 것 같아요!